.gitignore

```diff
@@ -17,7 +17,6 @@ node_modules
 .vscode/**/*
 !.vscode/launch.json
 !.vscode/tasks.json
```

Comment:
Can we remove also tasks and launch @matejkriz ?

Reply:
yes please, remove it

Reply:
changed with a313dafa7